packages/sdk/src/sudoswap/order.ts

```diff
@@ -27,3 +27,13 @@ const normalize = (order: Types.OrderParams): Types.OrderParams => {
     price: s(order.price),
   };
 };
+
+export class SwapList {
```

Comment:
Feels cleaner and more consistent to just use the `SwapList` type from the `types.ts` filed instead of having a class for it.